docs/book/basics/simple_resource.md

```diff
@@ -68,7 +68,7 @@ field, which may be set by autoscalers.
 ```go
 // ContainerSetSpec defines the desired state of ContainerSet
 type ContainerSetSpec struct {
-  // replics is the number of replicas to maintain
+  // replicas is the number of replicas to maintain
```

Comment:
It should match the field name, i.e. Replicas

Reply:
I think doc is referencing to JSON representation, so lowercase: just take a look:

```
  // **image** is the container image to run.  Image must have a tag.
  // +kubebuilder:validation:Pattern=.+:.+
  Image string `json:"**image**,omitempty"
```